### ISSUE TITLE
Version 4.9.40

### ISSUE THREAD

**Issue by Gusted** [darkreader/darkreader#7207]
*Gusted give Gusted a bypass to not follow the issue template.*

Hi @alexanderby, @bershanskiy and everyone whose interested into the development of Dark Reader!

This issue serves as "discussion" to discuss the things that need to be done before v4.9.40 can be released.
This should prevent that the time between releases becomes too long and that the amount of regression is a huge pile(as seen in our v4.9.36-v4.9.39 releases), I would like to set the release date on next week on Tuesday(2 November 2021).

I currently don't have any release-blockers to prevent a release on next Tuesday, I only have a lot of arguments that it should be happening on that date, given that we've fixed 2 pretty annoying regression(#7049, #7026) in this release and done a lot of work towards optimizations and MV3 compatibility(Thanks @bershanskiy!).

If someone thinks the release should be halted or moved to a further date, please feel free to speak up, otherwise be silent for a eternity.

Regards,
Gusted

**Comment by bershanskiy** [comment]
There are no release blockers on my side. Looking forward to the release on Tuesday!

**Comment by Gusted** [comment]
Sorry for the late update - the update was postponed to today because of some last-minute fixes. The update is currently being published to the extensions stores give it a few days before it's public accessible. Closing.
